core/src/com/po/fuck/weapons/Bullet.java

```diff
@@ -29,6 +29,9 @@ public void draw(CenterDrawer drawer) {
         sprite.setRotation(-velocity.angleDeg());
         drawer.draw(sprite, position);
     }
+    protected boolean tryDamage(Entity entity) {
```

Comment:
This function should damage enemy, if you only want to check it should be name `canDamage`.

Reply:
Also we need to add some damage(maybe DPS, because we have a laser gun) parameter to our bullet